src/mudlet.h

```diff
@@ -723,6 +725,9 @@ private slots:
     // Stores the translated names for the Encodings for the static and thus
     // const TBuffer::csmEncodingTable:
     QMap<QByteArray, QString> mEncodingNameMap;
+
+    // Whether multi-view is in effect (if more than one tab is present)
```

Comment:
Not sure if this is the right place to explain multi-view elementals
```suggestion
    // Whether multi-view is in effect
```

Reply:
Explaining what a flag does/controls/indicates? This is one of the best places for it as in a search you can always pick out (and in fact IDEs include the Qt one home in on) the place something is ***declared***.

Reply:
Yes `mMultiView` indicates "Whether multi-view is in effect"
That's fair, that is my suggestion as well.
Whereas "Multi-view is only possible if more than one tab is present" - has nothing to do with mMultiView.

Reply:
:thinking: *well* I sort of see what you are saying - and in practice this flag will be reflected in the up/down state of the toolbar button and the unchecked/checked state of the menu item. Whether those controls are *enabled* is a separate issue (which is tied to the count of tabs present as it happens, now!) and if multi-view was present when the user gets back to only having one profile open, the controls will stay (locked) in their up/down/unchecked/checked state and will not be alterable until a second profile is active again...